### ISSUE TITLE
Scale (and possibly also offset) not applied to MOU decision variables with "fixed" PARTRANS when supplied in an mou_dv_population_file during generation 0

### ISSUE THREAD

**Issue by ntdosch** [usgs/pestpp#227]
We've been having trouble getting the correct sign on DV pumping rates with "fixed" partrans during generation 0 when we supply a mou dv population file. Pumping rates in the initial dv pop file are are all positive values. When MOU builds the MODFLOW well package from our template, we end up with positive rates for wells with "fixed" partrans and negative rates for wells with "none" partrans. Here is a summary from a random member of generation 0 (first six rows are "fixed", second six rows are "none"): 
![gen0](https://user-images.githubusercontent.com/25257695/213813650-840a2a09-cefc-42e3-8543-9eda78864f01.png)

Is it possible that the scale multiplier is not applied to "fixed" values during gen 0? This issue seems to be resolved during generation 1 when MOU generates the dv population (though pumping rates in _case_.1.dv_pop.csv are also all positive). Here is the same table from a random member of gen 1 (all well package q's are negative):

![gen1](https://user-images.githubusercontent.com/25257695/213813674-fb94a667-4e6a-4182-8f7f-ae6159699ad8.png)

**Comment by jtwhite79** [comment]
I'd say it's highly probable that scale and offset are not applied to fixed dvs.   I'll have a look at this asap.

**Comment by jtwhite79** [comment]
Ok I see what is happening here its actually by design.  If you pass values in an existing population/ensemble file for fixed parameters, that signals mou (and ies and da) that you know what you are doing and you want to use the values in the existing population/ensemble exactly as they are - this is a way to include stochastic but fixed parameters (say for future recharge).  So in your case, by leaving those fixed parameter columns in the population file, mou is using the values "as-is" for each population member, but then when new members are generated, they get the ctrl file fixed parameter values (which now include scale and offset) - yikes! So to get around this, its probably easiest to just leave the fix parameters out of the population file you are passing in (which will trigger mou to use the ctrl file value and scale and offset for these fixed parameters).  And I'll add some traps for this condition (fixed pars being passed but with non-trivial scale/offset).  Is that a good compromise here?

**Comment by ntdosch** [comment]
Thanks @jtwhite79, that reasoning makes sense and your solution sounds super reasonable for this (kinda weird) edge case. Really appreciate your quick response!

**Comment by mnfienen** [comment]
Hey guys - this makes sense and explains what we were running into. The thing that was most confusing for us was that we were getting scale applied (e.g. flip in sign) in some DVs but not others. So, as we move parameters in and out of fixed status for experiments, we had to also change sign (which also means reversing upper and lower bounds). It gets hella confusing. I guess leaving fixed pars completely out of initial DV population is the ticket. 

One alternative thought is that we have been regarding DVs as parameters defined in the PST control file, not necessarily raw model input parameters, so it seems a bit inconsistent to treat them differently if fixed or not when they are all supplied in a single column in the initial population. That said, we solved the issue based on your feedback and can keep it rolling. Also as Nick said, this is kinda an edge case.
